src/Stoa.ts

```diff
@@ -2580,6 +2582,51 @@ class Stoa extends WebService {
             });
     }
 
+    /* Get validator reward
+     * @returns Returns reward of the validators.
+     */
+    public async getValidatorReward(req: express.Request, res: express.Response) {
+        const address = String(req.params.address);
+
+        let validatorAddress: PublicKey;
+        try {
+            validatorAddress = new PublicKey(address);
+        } catch (error) {
+            res.status(400).send(`Invalid value for parameter 'address': ${address}`);
+            return;
+        }
+        const pagination: IPagination = await this.paginate(req, res);
+        this.ledger_storage
+            .getValidatorReward(address, pagination.pageSize, pagination.page)
+            .then((data: any[]) => {
+                if (data.length === 0) {
+                    return res.status(204).send(`The data does not exist.`);
+                } else {
+                    let rewards: IValidatorReward[] = []
+                    for (const row of data) {
+                        rewards.push({
+                            block_height: row.block_height,
+                            freezing_amount: row.type === 1 ? row.amount : 0,
```

Comment:
Freezing_amount is actually the amount that the node has staked.
This is the UTXO amount of enrollment.